test/test_rostopic/test/test_rostopic_unit.py

```diff
@@ -388,3 +388,45 @@ def test_create_field_filter(self):
         self.assertEquals(['t', 'd'], list(f(m)))
         m = Header()
         self.assertEquals(['seq', 'stamp'], list(f(m)))
+
+    def test_slicing(self):
+        from test_rospy.msg import ArrayVal, Val
```

Comment:
The `test_rostopic` package does not depend on `test_rospy`. Please add the necessary messages to `test_rostopic` and use them from there.